src/components/Experience.svelte

```diff
@@ -13,10 +16,20 @@
   export let currentIndex: number
 
   let events: Event[]
+  let userDoc: DocumentReference<DocumentData>
+  api.currentUserDoc((v) => { userDoc = v })
+
+  const sendResults = () => {
+    const experimentResult: Experiment = {
+      id: `${displayChunkLength}-${inputChunkLength}`,
+      events
+    }
+    api.addExperimentRequest(userDoc, experimentResult)
+  }
```

Comment:
♻️ Move all this logic to `addExperimentRequest` directly. There shouldn' t be an import from firstore in this component

Reply:
✅